### PR TITLE
Remove os system cmd

### PR DESCRIPTION
**Pull request by spborder**
Fixes spaces/odd characters in file names issue. Creates importable "run_it" function. Updates names of some arguments for consistency.

### CODE REVIEW

File: multic/cli/MultiCompartmentSegment/MultiCompartmentSegment.py [hunk 1]
```diff
@@ -1,23 +1,102 @@
...
+        'imBoxExt':'.jpeg',
+        'finalImgExt':'.jpeg',
+        'wsi_ext':'.svs,.scn,.ndpi',
+        'bg_intensity':0.5,
+        'approx_downsample':1,
+        'min_size':[30,30,30,30,30,30],
+        'bordercrop':300,
+        'LR_region_pad':50,
+        'show_interstitium':True
+    }
```
Comment (sayatmimar) [inline multic/cli/MultiCompartmentSegment/MultiCompartmentSegment.py:90]: We can move this outside of the function to the top. You can also call it `DEFAULT_VALS`. As default `args` are now stored in this dictionary, we don't need to parse them in `segmentation_school` right? If so let's delete parsing in `segmentation_school`.

**Review by sayatmimar — CHANGES_REQUESTED** [review]
See my comment. Have you built the docker and tested this code in DSA?

**Comment by spborder** [comment]
Fixed, updated image, tested on EC2 and all looks good.

Completed Job: http://ec2-3-230-122-132.compute-1.amazonaws.com:8080/#job/65de6cd8adb89a58fea10d7c
Test Image: http://ec2-3-230-122-132.compute-1.amazonaws.com:8080/#item/659c269b4e590bbcdeb406cc

**Review by sayatmimar — APPROVED** [review]
LGTM

**Pull request merged**